src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,28 @@
+package main.java.duke;
+import java.util.ArrayList;
+
+import main.java.duke.tasks.Task;
+
+public class TaskList {
+    protected ArrayList<Task> taskList;
+    /**
+     * Constructs a new task list with a given list of tasks
+     *
+     * @param taskList a list of tasks
+     */
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Constructs a new empty task list
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public ArrayList<Task> getTaskList() {
```

Comment:
Perhaps this class could have additional functionality to operate on the ArrayList and avoid having to return it?